tn40.c

```diff
@@ -2857,7 +2859,7 @@ static int bdx_get_phy_by_id(int vendor, int device, int subsystem)
 		    )
 			return bdx_dev_tbl[i].phy_type;
 	}
-	return 0;
+	return PHY_TYPE_NA;
 }
```

Comment:
Change return type of the function to `enum PHY_TYPE`. (minor, doesn't hinder merge)

Reply:
Thank you for the feedback. I did a commit to the dame branch and it looks like that included it in the same pull request automatically. (still finding my way around git and github) Please let me know if there are other issues or comments.

Reply:
Thanks, it looks good now. Another minor code style issue is a space before the first `{` in the if body.

I don't have merge rights, so we'll have to wait until someone that has, comes around.